.github/workflows/deploy.yaml

```diff
@@ -88,10 +88,47 @@ jobs:
         env:
           HOPR_PACKAGE: hoprd
 
+  deploy_smart_contracts:
+    name: Deploy Smart Contracts
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Configure Git info
+        run: bash -x scripts/configure-git-info.sh
+
+      - name: Update repo from latest pushes.
+        run: |
+          git pull origin ${{ github.ref }} # This should pull new packages with versions etc.
+
+      - name: Deploy 
+        working-directory: packages/ethereum
+        env:
+          PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
+          QUIKNODE_KEY: ${{ secrets.QUIKNODE_KEY }}
+        run: /
+          npx hardhat deploy --network xdai
```

Comment:
Can we move this into a script and make it configurable in regards to the network used?